### PR TITLE
qt: Fix locating QtWebEngineProcess executable

### LINKED ISSUE #186: [Qt] Support QTWEBENGINEPROCESS_PATH
We should consider adding this in our Qt launcher.

Reference: https://doc.qt.io/qt-5/qtwebengine-deploying.html#deploying-qt-webengine-processes

### PR DESCRIPTION
**Pull request by Lin-Buo-Ren**
Fix Qt Web Engine application by setting the proper
QTWEBENGINEPROCESS_PATH.

Fixes #186.

Signed-off-by: 林博仁(Buo-ren, Lin) <Buo.Ren.Lin@gmail.com>

### CODE REVIEW

**Comment by pachulo** [comment]
I think that this error I encountered snapping Musescore 3.0 might be related:

 - https://musescore.org/en/comment/884393#comment-884393

File: qt/launcher-specific [hunk 1]
```diff
@@ -56,3 +56,5 @@ fi
 # Use GTK styling for running under the GNOME desktop
 append_dir GTK_PATH $RUNTIME/usr/lib/$ARCH/gtk-2.0
 
+# Fix locating the QtWebEngineProcess executable
+export QTWEBENGINEPROCESS_PATH=$RUNTIME/usr/lib/$ARCH/qt5/libexec
```
[the remark this answers is not in the document]
Comment (Lin-Buo-Ren) [reply qt/launcher-specific]: Thanks for pointing out, will update the patch shortly after work.

**Comment by pachulo** [comment]
Any news on this? Thanks!

File: qt/launcher-specific (hunk 1 above)
[the remark this answers is not in the document]
Comment (oSoMoN) [reply qt/launcher-specific]: @Lin-Buo-Ren it looks like this is being used successfully [there](https://forum.snapcraft.io/t/problems-snapping-new-nextcloud-desktop-sync-client/7757/19), so I'm happy to approve and merge as soon as you fix the path.

**Comment by Lin-Buo-Ren** [comment]
Apologies for the delay, (hopefully) I'll make the change in the next 10 hours.

**Comment by Lin-Buo-Ren** [comment]
I've applied the requested change, please review.

**Pull request merged**